meson.build

```diff
@@ -77,6 +77,10 @@ nntrainer_conf.set('EXEC_PREFIX', nntrainer_bindir)
 nntrainer_conf.set('LIB_INSTALL_DIR', nntrainer_libdir)
 nntrainer_conf.set('INCLUDE_INSTALL_DIR', nntrainer_includedir)
 
+dummy_dep = dependency('__dummy__', required: false)
+openmp_dep = dependency('openmp')
+
+blas_dep = dummy_dep
```

Comment:
Well.. I think it's not a good way. If you wanna use an empty dependency, just use `blas_dep = declare_dependency()`

Reply:
`dep = declare_dependency()` returns `dep.found() == true`
https://github.com/mesonbuild/meson/issues/2872#issuecomment-355252844

to make a not found it is recommend to use `dependency('', required: false)`
https://github.com/mesonbuild/meson/pull/2615/commits/b573b8f59738f54a6d630589870563b8dd52cb0c

IMO, dummy dep in this case should return `dep.found() == false`
So I changed accordingly.
Thanks for the suggestion!

Reply:
Hmm.. I think `dep.found()` would be never used in the current meson script.

Anyway, now looks better than before! FYI, my concern was that the build script was generating the unnecessary log about `__DUMMY__`.